internal/translation/customroles/conversion.go

```diff
@@ -0,0 +1,127 @@
+package customroles
+
+import (
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/pointer"
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+)
+
+type CustomRole struct {
+	*akov2.CustomRole
+}
```

Comment:
Why wrapping if we are not adding anything?

Reply:
TBH I was just following what was done in the Auditing translation layer. I suppose this gives us flexibility to add more fields in future if we ever see the need? I can change to use the akov2 type if preferred.

Reply:
Ok, let's leave it for regularity and expandability then.